inductiva/tasks/task.py

```diff
@@ -401,16 +414,56 @@ def download_outputs(
             warnings.warn("Path already exists, files may be overwritten.")
         output_dir.mkdir(parents=True, exist_ok=True)
 
+        download_message = "Downloading simulation outputs to %s"
```

Comment:
on line 408, there is no need to make the same call on each branch of the if statement. You can do:

```python
if output_dir is None:
   output_dir = self.id

output_dir = files.resolve_output_path(output_dir)
```

It could even just be (but I don't necessarily recommend it as it might be harder to read):

```python
output_dir = files.resolve_output_path(output_dir or self.id)
```

Reply:
Thanks!